config/default.json

```diff
@@ -71,6 +71,7 @@
         "RestrictPublicChannelDeletion": "all",
         "RestrictPrivateChannelDeletion": "all",
         "RestrictPrivateChannelManageMembers": "all",
+        "EnableXToLeaveChannelsFromLHS": false,
```

Comment:
Shouldn't this be a user setting and not a system setting?

Reply:
I mean that each user should be able to enable and disable it individually. It's not really a system wide decision.

Reply:
@crspeller  I don't think this needs to be a user setting. It would be weird if some users can hit "x" to leave a channel, but other users on the system can't.

As a note, this setting is "experimental". We're working on updating the channel model, which might let you hide channels without actually leaving them, similar to DMs. Once we've made a decision on that, this setting might get removed or replaced by something else.